react_admin/src/container/App.js

```diff
@@ -19,6 +19,7 @@ import ReferVolunteer from '../adminWebsite/ReferVolunteer';
 import ConfirmatinScreen from '../adminWebsite/confirmationScreen';
 import TrustMemberScreen from '../adminWebsite/TrustMemberScreen';
 import AdminUploadDEOresponse from '../adminWebsite/adminUploadDEOresponse';
+import DEOEmailTrigger from '../adminWebsite/emailDEOTrigger';
```

Comment:
@kamalkanth-dev , I've updated this file and checked-in y'day (pending merge). Once it is merged I will merge yours. We may run into conflict issues but we can resolve before this weekend. Thanks!

Reply:
@kamalkanth-dev , when time permits help resolve this conflict. thanks!